objects/obj_al_ship/Alarm_0.gml

```diff
@@ -1,292 +1,888 @@
-
-
+// Sets up the ships for each race
 cooldown[1]=0;
 cooldown[2]=0;
 cooldown[3]=0;
 cooldown[4]=0;
 cooldown[5]=0;
```

Comment:
wrap in loop to make easier to augment 
```
for (var i =0;i<array_length(cooldown);i++){
      cooldown[i] = 0;
}
``` may need to use a hard limit rather than array length depending on how the array cooldown works